canonicalwebteam/blog/blog_api.py

```diff
@@ -177,3 +202,33 @@ def _get_month_name(self, month_index):
         """
 
         return date(1900, month_index, 1).strftime("%B")
+
+    def _apply_image_template(self, content, width, height=None):
+        """ Apply image template to the img tags
+
+        :param content: String to replace url
+        :param width: Default width of the image
+        :param height: Default height of the image
+
+        :returns: HTML images templated
+        """
+
+        soup = BeautifulSoup(content, "html.parser")
+        for image in soup.findAll("img"):
+            img_width = image.get("width")
+            img_height = image.get("height")
+
+            new_image = BeautifulSoup(
+                image_template(
+                    url=image.get("src"),
+                    alt="",
+                    width=img_width if img_width is not None else width,
```

Comment:
couldn't this be simply:

```suggestion
                    width=img_width or width,
```

?